src/veranda/errors.py

```diff
@@ -0,0 +1,248 @@
+# # Copyright (c) 2019, Vienna University of Technology (TU Wien), Department of
```

Comment:
Why is everthing commented?

Reply:
Yes, because it is not intended for usage - this should be a simple development release. I think this is safer than people starting to use things, which should not be used yet.